### ISSUE TITLE
The binary from the release page cannot execute `sqlite3` command

### ISSUE THREAD

**Issue by autopp** [sue445/plant_erd#155]
I download `plant_erd_darwin_amd64` from [the release page](https://github.com/sue445/plant_erd/releases/tag/v0.2.2).

When `sqlite3` command is given, `plant_erd` exits with error.

```
$ plant_erd sqlite3 --database erd_test.sqlite3
2021/08/19 20:45:02 Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requires cgo to work. This is a stub

$ echo $?
1
```

**Comment by sue445** [comment]
@autopp Does [0.2.1](https://github.com/sue445/plant_erd/releases/tag/v0.2.1) works?

**Comment by autopp** [comment]
Thanks, v0.2.1 was works. I'll use v0.2.1 until this issue is fixed.

**Comment by sue445** [comment]
@autopp Sorry, this is a regression of v0.2.2. 

There are no functional differences regarding SQLite3 between v0.2.1 and v0.2.2
https://github.com/sue445/plant_erd/blob/master/CHANGELOG.md#v022

Please use v0.2.1 until I fix this 🙏

**Comment by sue445** [comment]
Fixed at v0.2.4
https://github.com/sue445/plant_erd/releases/tag/v0.2.4
